Sources/Purchasing/ProductsManager.swift

```diff
@@ -126,6 +135,25 @@ private extension ProductsManager {
         return self.productsFetcherSK1.products(withIdentifiers: identifiers, completion: completion)
     }
 
+    func trackProductRequestIfNeeded(_ startTime: Date,
```

Comment:
I think this should be:

```suggestion
    func trackProductsRequestIfNeeded(_ startTime: Date,
```

Reply:
Addressed in https://github.com/RevenueCat/purchases-ios/pull/4247/commits/ea41bf72dc1b301b66a7cbdf02e1be8c01e3ccc8